recipes/ctpg/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class DawHeaderLibrariesTestConan(ConanFile):
```

Comment:
Copy past detected!
Copyright lawyers will soon arrive at your door!
```suggestion
from conans import ConanFile, CMake, tools


class TestPackageConan(ConanFile):
```

Reply:
Sorry my mistake.